### PR TITLE
Azure AD authentication with new Microsoft Graph API

### PR DESCRIPTION
**Pull request by KannappanSomu**
Closes https://github.com/influxdata/chronograf/issues/5864

_Describe your proposed changes here._

Added documentation for Azure AD authentication with new Microsoft Graph API as the old azure AD graph API is deprecating in June 2022.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by KannappanSomu** [comment]
!signed-cla

**Comment by KannappanSomu** [comment]
!signed-cla

File: content/chronograf/v1.9/administration/managing-security.md [hunk 1]
```diff
@@ -394,6 +397,42 @@ export HEROKU_ORGS=hill-valley-preservation-sociey,the-pinheads
...
+   For example, if Chronograf is configured with a `GENERIC_NAME` value of AzureAD, the reply URL would be `http://localhost:8888/oauth/AzureAD/callback`.
+
+3. Your Azure AD application should have below permissions to read user profile.
+
+![image](https://user-images.githubusercontent.com/55806176/154274246-54ad08dd-2f8d-4f2c-95bc-a8c61dd798c5.png)
+
+4. After completing the application provisioning within Azure AD, you can now complete the configuration with Chronograf.
+   Using the metadata from your Azure AD instance, proceed to export the following environment variables:
+
+    Set the following environment variables in `/etc/default/chronograf`:
```
Comment (kelseiv) [inline content/chronograf/v1.9/administration/managing-security.md:418]:
```suggestion
4. To update the Chronograf configuration, use your Azure AD instance metadata
to set the following environment variables in `/etc/default/chronograf`:
```

File: content/chronograf/v1.9/administration/managing-security.md [hunk 2]
```diff
@@ -394,6 +397,42 @@ export HEROKU_ORGS=hill-valley-preservation-sociey,the-pinheads
...
+    GENERIC_SCOPES=openid
+    GENERIC_CLIENT_ID=<<APPLICATION-ID>>
+    GENERIC_AUTH_URL=https://login.microsoftonline.com/<<TENANT-ID>>/oauth2/authorize?resource=https://graph.microsoft.com
+    GENERIC_CLIENT_SECRET=<<APPLICATION-KEY>>
+    TOKEN_SECRET=secret
+    GENERIC_API_URL=https://graph.microsoft.com/v1.0/me
+    PUBLIC_URL=http://localhost:8888
+    ```
+
+    Note: If you’ve configured TLS/SSL, modify the `PUBLIC_URL` to ensure you're using HTTPS.
```
Comment (kelseiv) [inline content/chronograf/v1.9/administration/managing-security.md:434]:
```suggestion
{{% note %}}
    **Note:** If you’ve configured TLS/SSL, modify the `PUBLIC_URL` to ensure you're using HTTPS.
{{% /note %}}
```

**Comment by kelseiv** [comment]
Thank you for the PR, @KannappanSomu! 
A couple suggestions, and we'll merge--sorry about the CLA request--will look into this issue.

**Review by kelseiv — APPROVED** [review]

**Pull request merged**